packages/app-store/index.ts

```diff
@@ -1,33 +1,35 @@
 const appStore = {
-  // example: import("./example"),
-  applecalendar: import("./applecalendar"),
-  caldavcalendar: import("./caldavcalendar"),
-  closecom: import("./closecom"),
-  dailyvideo: import("./dailyvideo"),
-  googlecalendar: import("./googlecalendar"),
-  googlevideo: import("./googlevideo"),
-  hubspot: import("./hubspot"),
-  huddle01video: import("./huddle01video"),
-  jitsivideo: import("./jitsivideo"),
-  larkcalendar: import("./larkcalendar"),
-  office365calendar: import("./office365calendar"),
-  office365video: import("./office365video"),
-  plausible: import("./plausible"),
-  salesforce: import("./salesforce"),
-  zohocrm: import("./zohocrm"),
-  sendgrid: import("./sendgrid"),
-  stripepayment: import("./stripepayment"),
-  tandemvideo: import("./tandemvideo"),
-  vital: import("./vital"),
-  zoomvideo: import("./zoomvideo"),
-  wipemycalother: import("./wipemycalother"),
-  giphy: import("./giphy"),
-  zapier: import("./zapier"),
-  exchange2013calendar: import("./exchange2013calendar"),
-  exchange2016calendar: import("./exchange2016calendar"),
-  exchangecalendar: import("./exchangecalendar"),
-  facetime: import("./facetime"),
-  sylapsvideo: import("./sylapsvideo"),
+  // example: (): Promise<typeof import("./example")> => import("./example"),
+  applecalendar: (): Promise<typeof import("./applecalendar")> => import("./applecalendar"),
+  caldavcalendar: (): Promise<typeof import("./caldavcalendar")> => import("./caldavcalendar"),
+  closecom: (): Promise<typeof import("./closecom")> => import("./closecom"),
+  dailyvideo: (): Promise<typeof import("./dailyvideo")> => import("./dailyvideo"),
+  googlecalendar: (): Promise<typeof import("./googlecalendar")> => import("./googlecalendar"),
+  googlevideo: (): Promise<typeof import("./googlevideo")> => import("./googlevideo"),
+  hubspot: (): Promise<typeof import("./hubspot")> => import("./hubspot"),
```

Comment:
This can also be
```suggestion
  hubspot: () => import("./hubspot"),
```

Reply:
> This can also be
yes, but the types are not being inferred, I tried that but got the same result, so yes.